src/ai/backend/manager/cli/__main__.py

```diff
@@ -16,6 +16,7 @@
 from redis.asyncio.client import Pipeline
 from setproctitle import setproctitle
 
+from ai.backend.cli.types import ExitCode
```

Comment:
If it is used in more than `ai.backend.cli`, how about relocating it to `ai.backend.common`?

Reply:
Following your opinion, I have two selective ideas.

1. Change  package module name
Default 
```console
from ai.backend.cli.types import ExitCode
```
Change all package module name from `ai.backend.cli.types.ExitCode` to `ai.backend.common.types.ExitCode`

Change
```console
from ai.backend.common.types import ExitCode
```


2. make a new Package 

Package module `ai.backend.cli.types.ExitCode`  already exists, but defines a new package `ai.backend.common.types.ExitCode` .

How about your opinion? @rapsealk  @achimnol  @Yaminyam @inureyes ?

Reply:
No, it should be in `ai.backend.cli`.
`ai.backend.common` will be gradually decomposed into multiple smaller packages.